deepchecks/vision/batch_wrapper.py

```diff
@@ -75,13 +90,19 @@ def __getitem__(self, index: int):
 
     def __len__(self):
         """Return length of batch."""
-        return len(self._batch)
+        dataset = self._context.get_data_by_kind(self._dataset_kind)
```

Comment:
@JKL98ISR Can you explain the logic of this code? Possible also in comments. 
Seems to me like this returns the number of batches in the dataset.